src/elements/featured-videos.html

```diff
@@ -166,8 +172,13 @@
             <div class="video-item">
               <paper-icon-button class="video-play" icon="hoverboard:play" on-tap="playVideo" flex layout horizontal center-center></paper-icon-button>
 
-              <plastic-image id="image[[index]]" class="preview-image" srcset="[[getSessionsImageUrl(block.thumbnail, index)]]" sizing="cover"
-                lazy-load preload fade></plastic-image>
+              <plastic-image
+               id="image[[index]]"
+               class="preview-image"
+               srcset="[[getSessionsImageUrl(block.thumbnail, index)]]"
+               sizing="cover"               
+               style="background-color: {$ webapp.themeColor $};"
```

Comment:
I would suggest to use '--secondary-background-color: #F5F5F5;' We have too big difference between webapp color and background
https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/elements/shared-styles.html#L28